ci/svd2rust-regress/src/svd_test.rs

```diff
@@ -215,12 +219,13 @@ pub fn test(
     let cargo_check_err_file = path_helper_base(&chip_dir, &["cargo-check.err.log"]);
     let output = Command::new("cargo")
         .arg("check")
+        .arg("--all-features")
         .current_dir(&chip_dir)
         .output()
         .chain_err(|| "failed to check")?;
     output.capture_outputs(
         true,
-        "cargo check",
+        "cargo check --all-features",
```

Comment:
don't us `--all-features`. Just itemize which you need. If we add any incompatible feature in the future this will be broken